lingvo/tools/wpm_encode_file.py

```diff
@@ -87,11 +89,9 @@ def _MakeTfExample(enc, src_i, src_s, tgt_i, tgt_s):
 
 
 def _Preprocess(text):
```

Comment:
Please reflect that in the change description

Reply:
Done but... The difference between __range(x)__ and __xrange(x)__ is insignificant when x < 5_000_000.  The programmer time lost in making the multiple times in the next 302 days is not worth it.

Reply:
Oh, it's not so much range vs xrange, but the pylint comment which is required to pass the tests.